### DISCUSSION TITLE
Webpack ModuleFederationPlugin

### DISCUSSION THREAD

**Discussion by ritbl** [grafana/plugin-tools#202] [General]
Hello, I am trying to use ModuleFederationPlugin with grafana app plugin. 
When lazy component is being resolved, `remotes` are not loaded correctly.

**Comment by ritbl** [comment]
Probably it will not work due to the fact of SystemJS is being used for plugin loading.

**Comment by jackw** [comment]
@ritbl As you've correctly identified we currently use SystemJS to load plugins in Grafana and offer no options for using Module Federation within a plugin right now I'm afraid.
